### PR TITLE
Ratings & Reviews PR

### PR DESCRIPTION
**Pull request by jordanvillacorta**
Major changes:

- Add average star rating functionality
- Add recommend percentage functionality
- Add conditionals to catch if there are no recommendations or if percentage is NaN.
- Add CSS file for future styling
- Add properties to RatingsBreakdown state
- Add functions to helperFunction

### CODE REVIEW

File: client/src/ratings_reviews/RatingsBreakdown.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      averageRating: '',
+      averageStarRating: '',
+      recommendPercentage: ''
+    }
+  }
+
+  componentDidUpdate(prevProps) {
```
Comment (SebasCC99) [inline client/src/ratings_reviews/RatingsBreakdown.jsx:18]: please fix the indentation of lines

File: client/src/ratings_reviews/ReviewsList.jsx [hunk 2]
```diff
@@ -1,37 +1,61 @@
...
     this.state = {
-      reviewsList: []
+      reviewsList: [],
+      sortOption: 'newest'
     }
-
+    // this.handleChange = this.handleChange.bind(this);
   }
 
   componentDidUpdate(prevProps) {
```
Comment (SebasCC99) [inline client/src/ratings_reviews/ReviewsList.jsx:17]: please fix the indentation

**Review by SebasCC99 — CHANGES_REQUESTED** [review]
Please map in reviews list, render a review tile for each review

File: client/src/ratings_reviews/RatingsBreakdown.jsx (hunk 1 above)
Comment (jordanvillacorta) [reply client/src/ratings_reviews/RatingsBreakdown.jsx:18]: Fixed!

File: client/src/ratings_reviews/ReviewsList.jsx (hunk 2 above)
Comment (jordanvillacorta) [reply client/src/ratings_reviews/ReviewsList.jsx:17]: Fixed!

**Review by SebasCC99 — APPROVED** [review]
Nice job! Looking more organized and clean!

**Pull request merged**